packages/stack/stack-operator/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM golang:1.20-alpine as build
+WORKDIR /app
+COPY . .
+RUN CGO_ENABLED=0 go build -o /operator
+
+FROM alpine/k8s:1.24.16
+WORKDIR /
+COPY --from=build /operator /
```

@Timex1854 Use this approch for installing kubectl 
```suggestion
COPY --from=build /operator /
COPY --from=alpine/k8s:1.24.16 /usr/bin/kubectl /usr/local/bin/kubectl
```